common/scripts/base.sh

```diff
@@ -99,8 +99,7 @@ function undeploy() {
 # $2 - service port
 function waitUntilServiceIsActive() {
   marathon_lb_host_ip=$(dcos marathon app show marathon-lb | $mesos_artifacts_home/common/scripts/get-marathon-lb-host.py)
```

Comment:
We are now using host ports for mysql dbs. This script works for now because marathon app adds the same host port as the service port for dbs. Shouldn't we use the host agent ip to check with dbs?